internal/db/read_writer.go

```diff
@@ -533,6 +536,22 @@ func (rw *Db) SearchWhere(ctx context.Context, resources any, where string, args
 	return nil
 }
 
+// Now returns the current transaction timestamp.
+func (rw *Db) Now(ctx context.Context) (time.Time, error) {
+	const op = "db.(*Db).Now"
+	rows, err := rw.Query(ctx, "select current_timestamp", nil)
```

Comment:
It might be good to provide a link to the [postgres docs](https://www.postgresql.org/docs/current/functions-datetime.html#FUNCTIONS-DATETIME-CURRENT) since they provide a good explanation of this and the alternatives. 

I would also note that we have tended to use `now()` elsewhere in the code so it might be good to do that here as well.